package.json

```diff
@@ -35,8 +33,6 @@
         "startTmuxServer": "node_modules/tmex/tmex dev \"yarn startTscServer\" \"yarn startLernaWatcher\" \"yarn startAdminServer\" \"yarn startViteServer\"",
         "startTscServer": "tsc -b -verbose -watch",
         "startViteServer": "vite dev",
-        "startWordpressPlugin": "cd ./wordpress/web/app/plugins/owid && yarn && yarn start",
-        "startSiteFront": "./devTools/vite/startVite.sh",
```

Comment:
this is still used by make up and make up.full. I'll revert it now to be able to continue testing, but feel free to amend if you had something else in mind.

Reply:
Thanks, that was a mistake :)